CHANGELOG.md

```diff
@@ -47,6 +47,10 @@ suffix.
 
 - Add configuration option for `migration_path` [#4190](https://github.com/solidusio/solidus/pull/4190) ([SuperGoodSoft](https://github.com/supergoodsoft/))
 
+### Frontend
+
+- SolidusFrontend is deprecated; use [SolidusStarterFrontend](https://github.com/solidusio/solidus_starter_frontend) instead. SolidusFrontend will be removed in a future version of Solidus.
```

Comment:
Again, maybe we can be cautious here and make it clearer that we're not recommending old storefronts built on top of `solidus_frontend` to replace it.